gta/map.py

```diff
@@ -0,0 +1,119 @@
+# GTA DragonFF - Blender scripts to edit basic GTA formats
+# Copyright (C) 2019  Parik
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+
+#######################################################
+
+# Direct port of OpenRW's LoaderIPL.cpp ( https://github.com/rwengine/openrw/blob/master/rwengine/src/loaders/LoaderIPL.cpp )
+# Seems to only support model INSTances and ZONEs. Though I believe the
+# INSTances alone should give us enough info to import the entire maps
+# of III / VC / SA.
+
+import os 
+from enum import Enum
+
+#######################################################
+class SectionTypes(Enum):
+	INST = 1
+	PICK = 2
+	CULL = 3
+	ZONE = 4
+	NONE = 5
+
+#######################################################
+def loadIPL(filepath):
```

Comment:
I would prefer a class over a function. This could then have members like `inst_list`, `pickup_list`, `cull_list`, and so on.